bin/pygac-run

```diff
@@ -90,19 +77,72 @@ def str2scanline(string):
 def validate_args(args):
     if args.end_line > 0 and args.start_line > args.end_line:
         raise ValueError('Start Scanline > End Scanline')
+    if args.config and not os.path.isfile(args.config):
+        raise FileNotFoundError(
+            'The provided config file "%s" does not exist!'
+            % args.config)
+
+def tarfile_walker(path):
+    """Yield all files from a tar archive.
+
+        Args:
+            path (str): Path to archive
+        Yields:
+            filename: name of the extracted file
+            fileobj: corresponding file object
+    """
+    with tarfile.open(path) as archive:
+        for tarinfo in archive:
+            if tarinfo.isfile():
+                filename = tarinfo.name
+                fileobj = archive.extractfile(filename)
+                yield filename, fileobj
+
+def logged_trial(processor):
+    """Decorator to logs exceptions during processing."""
+    @wraps(processor)
+    def wrapper(filename, *args, **kwargs):
+        try:
+            processor(filename, *args, **kwargs)
+        except Exception:
+            logger.exception('Could not porcess "%s"' % str(filename))
```

Comment:
Now, the exceptions are not caught for single file execution.

Reply:
@carloshorn Regarding the `debug` option I was thinking of something like this (from the top of my head):

```python
def wrapper(filename, debug, *args, **kwargs):
    try:
        processor(filename, *args, **kwargs)
    except Exception:
        if debug:
            raise
        else:
            logger.exception(...)
```

This could then be called like `processor(filename, args.debug, start_line, end_line)`

Reply:
Oh, and there's a little typo in the log message: `porcess` vs `process`